pkg/utils/rebase_todo_test.go

```diff
@@ -284,7 +284,6 @@ func TestRebaseCommands_moveFixupCommitDown(t *testing.T) {
 			expectedErr: nil,
 		},
 		{
-			// TODO: is this something we actually want to support?
```

Comment:
I think this comment is outdated. This is testing `moveFixupCommitDown` and this is the only test case that actually moves a fixup commit down...

Comment added in https://github.com/jesseduffield/lazygit/commit/e63858215e8778f58d66bf064359d70a48eb6756

Reply:
"Outdated" is probably the wrong word, it was never accurate. I agree that we should simply remove it, can you add that to your branch?

Reply:
It was removed, I was just trying to go through my reasoning in this comment.

Reply:
Oops, I didn't really pay attention, sorry.